lib/core/services/alfresco-api.service.ts

```diff
@@ -41,8 +41,7 @@ export class AlfrescoApiService {
      */
     nodeUpdated = new Subject<Node>();
 
-    protected alfrescoApiInitializedSubject: BehaviorSubject<any>;
-    alfrescoApiInitialized: Observable<any>;
+    alfrescoApiInitialized: ReplaySubject<boolean> = new ReplaySubject(1);
```

Comment:
This way you are exposing the whole subject, making it accessible to modify from outside, which basically let other consumers of this class modify its behaviour. Is it intentional? I assume, it would be safer to expose only an observable interface of it with .asObservable() as it was before, otherwise it is easier to misuse it.